### PR TITLE
New version: Microsoft.WingetCreate version 1.2.6.0

### PR DESCRIPTION
**Pull request by mdanish-kh**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107856)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
While during deploying using Add-AppxPackage -Path "C:\Users\Brandon Wan\Downloads\Adk Install Microsoft.WindowsPackageManagerManifestCreator_1.2.6.0_8wekyb3d8bbwe.msixbundle" 
The Package Family Name is Microsoft.WindowsPackageManagerManifestCreator_8wekyb3d8bbwe

**Comment by mohammadhasangolzar** [comment]
thanks for adding new version to the repository

File: manifests/m/Microsoft/WingetCreate/1.2.6.0/Microsoft.WingetCreate.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Created with YamlCreate.ps1 v2.2.6 $debug=QUSU.CRLF.7-3-4.Win32NT
```
Comment (jedieaston) [inline manifests/m/Microsoft/WingetCreate/1.2.6.0/Microsoft.WingetCreate.installer.yaml:1]: Ironic.

**Review by jedieaston — APPROVED** [review]

File: manifests/m/Microsoft/WingetCreate/1.2.6.0/Microsoft.WingetCreate.installer.yaml (hunk 1 above)
Comment (mdanish-kh) [reply manifests/m/Microsoft/WingetCreate/1.2.6.0/Microsoft.WingetCreate.installer.yaml:1]: In my defence I did try creating it with winget-create first but there are still some bugs that need to be squashed there. 

Also unresolved review comments block merge these days..😄

**Pull request merged**